src/sass/partials/components/_nav.scss

```diff
@@ -41,3 +41,47 @@
     }
   }
 }
+
+.breadcrumb {
```

Comment:
This should sit in a separate file, `components/_breadcrumb`. While it is a form of navigation, each component file (`_base` excluded) should contain only one file, and it should be the same name as the component inside